src/hud.js

```diff
@@ -219,7 +226,7 @@ class HUD {
     // this function deteriorates the nutrition
     // and the hydration of the player.
     live() {
-        // for now we just deteriorate by a random ish 
+        // for now we just deteriorate by a random ish
```

Comment:
This also looks like old code [see new code for reference](https://github.com/felixangell/biodefence/blob/master/src/hud.js#L221)

It might be easier to create a new branch and manually move over your clock changes